### DISCUSSION TITLE
Code cleanup

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#6] [Ideas]
Should more template code be included?

I suggest keeping only the base code here, the api code should not be comprehensive

**Comment by downdawn** [comment]
(The display has been broken for the past two days)

I think the basic permission module can be added.

I even hope that it can be made like [go-admin](https://github.com/go-admin-team/go-admin) (also based on Casbin), because I find that there are few similar projects on the python web, which is more intuitive and can attract more people.

**Comment by wu-clan** [comment]
#11
